curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/use-typeof-to-check-the-type-of-a-variable.md

```diff
@@ -21,7 +21,7 @@ console.log(typeof {});
 
 In order, the console will display the strings `string`, `number`, `object`, and `object`.
 
-JavaScript recognizes six primitive (immutable) data types: `Boolean`, `Null`, `Undefined`, `Number`, `String`, and `Symbol` (new with ES6) and one type for mutable items: `Object`. Note that in JavaScript, arrays are technically a type of object.
+JavaScript recognizes seven primitive (immutable) data types: `Boolean`, `Null`, `Undefined`, `Number`, `String`, `BigInt` and `Symbol` (new with ES6) and one type for mutable items: `Object`. Note that in JavaScript, arrays are technically a type of object.
```

Comment:
BigInt was added with ES2020, it's more recent than Symbol, so or I would add that specification too or just remove the "new with ES6"

```suggestion
JavaScript recognizes seven primitive (immutable) data types: `Boolean`, `Null`, `Undefined`, `Number`, `String`, `Symbol` (new with ES6), and `BigInt` (new with ES2020), and one type for mutable items: `Object`. Note that in JavaScript, arrays are technically a type of object.
```

```suggestion
JavaScript recognizes seven primitive (immutable) data types: `Boolean`, `Null`, `Undefined`, `Number`, `String`, `BigInt` and `Symbol` and one type for mutable items: `Object`. Note that in JavaScript, arrays are technically a type of object.
```